eland/operations.py

```diff
@@ -279,11 +282,18 @@ def _metric_aggs(
         if numeric_only:
             # Consider if field is Int/Float/Bool
             fields = [field for field in fields if (field.is_numeric or field.is_bool)]
+        elif not numeric_only and (pd_aggs == ["quantile"]):
```

Comment:
Are there situations where pd_aggs has more than one aggregation including quantile? Like quantile and max? Basically asking if we should do `"quantile" in pd_aggs`?

Reply:
I'm thinking like when we want to use percentiles within the DataFrame.info() API I believe it's used along with other aggregations

Reply:
yeah, It was unreachable code anyway. Quantiles are working with timestamps, We filter out text fields in `is_es_agg_compatible` so, this is not required, Removing it.